app/scripts/views/force_auth.js

```diff
@@ -43,7 +43,7 @@ define(function (require, exports, module) {
       return {
         email: email,
         fatalError: getFatalErrorMessage(this, fatalError),
-        isPasswordAutoCompleteDisabled: this.isPasswordAutoCompleteDisabled(),
+       // isPasswordAutoCompleteDisabled: this.isPasswordAutoCompleteDisabled(),
```

Comment:
@shane-tomlinson  If I remove the comma here, it will produce an error, since its is not the end of the return